luigi_pipeline/seqr_loading.py

```diff
@@ -109,6 +139,14 @@ def validate_mt(mt, genome_version, sample_type):
         :param sample_type: WGS or WES
         :return: True or Exception
         """
+        if genome_version == '37':
+            contig_check_result = contig_check(mt, GRCh37_STANDARD_CONTIGS, VARIANT_THRESHOLD)
+        elif genome_version == '38':
+            contig_check_result = contig_check(mt, GRCh38_STANDARD_CONTIGS, VARIANT_THRESHOLD)
+
+        if not contig_check_result:
+            raise SeqrValidationError('The VCF failed contig check')
```

Comment:
this is kind of an unhelpful failure message - you go to all the trouble of figuring out why its failing in detail, and then all you include in the failure is just that it failed. The user will have to go digging through the logs to figure out what happened instead of getting a clear message in the job failure. `contig_check_result` should include information about what failed - it can even be a formatted error message if you want, or a list of errors - and then the exception you raise should include that information. At a high level, this message should tell you which if any chromosomes are missing completely, which if any are are below threshhold, and which if any are extra